### PR TITLE
Fix test failures introduced with the latest release

### PR DESCRIPTION
**Pull request by angrybayblade**
## Proposed changes

TBD

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes and CI passes too
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: aea/helpers/profiling.py [hunk 1]
```diff
@@ -46,22 +46,27 @@
 _default_logger = logging.getLogger(__file__)
 
 if platform.system() == "Windows":  # pragma: nocover
-    import win32process  # type: ignore  # pylint: disable=import-error
 
     WIN32_PROCESS_TIMES_TICKS_PER_SECOND = 1e7
 
     def get_current_process_memory_usage() -> float:
         """Get current process memory usage in MB."""
+        import win32process  # type: ignore  # pylint: disable=import-error,import-outside-toplevel
```
Comment (DavidMinarsch) [inline aea/helpers/profiling.py:54]: Suspect also need to import `win32timezone` - based on logs

Comment (angrybayblade) [reply aea/helpers/profiling.py]: Fixed here https://github.com/valory-xyz/open-aea/pull/298/commits/08a34f52efc732f3e39309562f67863e8cf711c0

File: aea/helpers/profiling.py [hunk 2]
```diff
@@ -47,26 +47,22 @@
 
 if platform.system() == "Windows":  # pragma: nocover
 
+    import win32process  # type: ignore  # pylint: disable=import-error,import-outside-toplevel
```
Comment (DavidMinarsch) [inline aea/helpers/profiling.py:50]: Why did you move it back out?

Comment (angrybayblade) [reply aea/helpers/profiling.py:50]: So this was wrong, the failure was not caused by this logic. It was a situation similar to missing `_strptime` module yesterday. The `win32process` import was not the issue, the issue was of missing `win32timezone` reference while importing `win32process` which was fixed in the latest commit

**Review by DavidMinarsch — APPROVED** [review]
LGTM
